OpenRA.Mods.Common/Widgets/Logic/Editor/MapEditorTabsLogic.cs

```diff
@@ -18,6 +18,7 @@ public class MapEditorTabsLogic : ChromeLogic
 	{
 		readonly Widget widget;
 
+		protected enum MenuType { Tiles, Layers, Actors, History }
```

Comment:
I suppose that wants to be in the first commit, otherwise I don't think that one would compile.